backend/service/k8s/cache.go

```diff
@@ -85,6 +91,59 @@ func (s *svc) startInformers(ctx context.Context, cs ContextClientset) {
 	s.topologyInformerLock.Release(topologyInformerLockId)
 }
 
+// cacheFullRelist will list all resources and push them to the topology cache for processing.
+// The importance of doing a semi frequent full re-list give us better cache accuracy,
+// while also keeping resources that are infrequently updated from being cleaned up by the cache TTL.
+// This works in tandem with the LightweightInformers above.
+//
+// Notably we intentionally run these in serial, not only can this cause memory pressure but
+// also being mindful of the kubernetes api servers to reduce burst load.
+func (s *svc) cacheFullRelist(ctx context.Context, lwPods, lwDeployments, lwHPA *cache.ListWatch) {
+	ticker := time.NewTicker(time.Hour * 1)
+	for {
+		// The informers will only ever do a full list once on boot
+		// we will wait the hour before doing another full list again
+		select {
+		case <-ticker.C:
+			pods, err := lwPods.List(metav1.ListOptions{})
+			if err != nil {
+				s.log.Warn("Unable to list pods to populate Kubernetes cache", zap.Error(err))
+				return
```

Comment:
why return? if pods list fails might deployments list succeed? also seems like we might want this at error severity, not a warn, given that there's no retry until the next run.

Reply:
Ah yeah I had these in parallel before let me remove the returns.